### DISCUSSION TITLE
Error: i.map is not a function

### DISCUSSION THREAD

**Discussion by PiotrEsse** [AntlerVC/firetable#919] [Support / Q&A] [answered]
👉 **Please describe how to reproduce this bug here.**
Ive just login, created workspace, project and try to connect to firebase test project. After authorisation - Ive got error message.

**Comment by BeeBombshell** [comment, marked as answer]
Hi @PiotrEsse. It's not very clear what error you're running into. Could you please send some screenshots or perhaps a video recording of what error you're facing so that we can help you out?

**Reply by BeeBombshell** [reply]
Hey, @PiotrEsse closing this discussion since it has been unattended for a long time. If you're still facing this issue, feel free to create a new one.
